.gitignore

```diff
@@ -91,7 +91,6 @@ c.out
 /cmd/weaviate-server/build_artifacts/
 
 esbackups/
-**/testdata/
```

Comment:
hmm, this might be a problem bc I think that our integration tests are using `testdata` folders and we don't want to those folders to be pushed, can we leave this change and rename your folder to `test/data` for example?
```
$ find . -type d -name testdata
./test/acceptance_with_python/.venv/lib/python3.12/site-packages/google/protobuf/testdata
./adapters/repos/db/lsmkv/testdata
./adapters/repos/modules/testdata
./.venv/lib/python3.12/site-packages/google/protobuf/testdata
```

Reply:
Noticed, we ignore `.venv` directory already on `.gitignore` so these cases are already ignored.
1. `./test/acceptance_with_python/.venv/lib/python3.12/site-packages/google/protobuf/testdata`
2. `./.venv/lib/python3.12/site-packages/google/protobuf/testdata`

After `test/run.sh -i` these are the only `testdata`
```
-bash-5.2$ find . -type d -name testdata
./.venv/lib/python3.12/site-packages/google/protobuf/testdata
./exp/query/testdata
```

And we will commit only `exp/query/testdata`.

Reply:
For the future record. We agreed on

1. Previously we globally ignore any `/testdata` folder, because some of our integration tests "generate" files on this directory and we don't to be pushed to our repo.
2. But at the same time, `testdata` is common convention in Go to put test related static files (that usually be commitable).
3. Proposal we agreed on.
   i. Change acceptance test script to use `generated_testdata` instead `testdata`
   ii. ignore `generated_testdata` instead globally